hack/lib/init.sh

```diff
@@ -140,6 +140,7 @@ KUBE_NONSERVER_GROUP_VERSIONS="
  imagepolicy.k8s.io/v1alpha1\
  admission.k8s.io/v1\
  admission.k8s.io/v1beta1\
+ namedresources.structured.resource.k8s.io/structured
```

Comment:
This is ....weird.  What is causing it?

Reply:
This is a left-over from defining "named resources" types in resource/structured/namedresources/v1alpha1. Code generation needed to know that this "group" was not served.

Removed.